<?xml version="1.0"?>
<launch>
  <node pkg="atl_ros" name="atl_camera" type="atl_pg_camera_node" output="screen" required="true">
    <!-- <param name="guid" type="str" value="0" />  <!&#45;&#45; first available camera &#45;&#45;> -->
    <param name="guid" type="str" value="49712223533654158" /> <!-- static camera -->
    <!-- <param name="guid" type="str" value="49712223533992369" /> <!&#45;&#45; gimbal camera &#45;&#45;> -->
    <param name="stamp_image" value="false" />
    <param name="image_topic" value="/atl/camera/image" />
    <param name="config_dir" value="$(find atl_configs)/configs/camera/pointgrey_firefly" />
  </node>
</launch>
